src/routes/message.route.js

```diff
@@ -0,0 +1,10 @@
+const { Router } = require('express');
+const messageController = require('../controllers/messanger.controller');
```

Comment:
There's a typo in the import statement: `messanger.controller` should be `messenger.controller` to match the correct spelling.